tracer/src/Datadog.Trace/Sampling/RateLimiter.cs

```diff
@@ -28,10 +28,10 @@ internal abstract class RateLimiter : IRateLimiter
         private int _previousWindowChecks = 0;
         private int _previousWindowAllowed = 0;
 
-        protected RateLimiter(int? maxTracesPerInterval)
+        protected RateLimiter(int? maxTracesPerInterval, int? intervalMilliseconds = null)
```

Comment:
Just wondering, would it be better to do this and avoid the nullability? 🤔 
```suggestion
        protected RateLimiter(int? maxTracesPerInterval, int intervalMilliseconds = 1_000)
```

If you're worried about duplicating the value, you can make it a `const` that the `TracerRateLimiter` can also use. My (tiny) concern is that it's not clear (at the call site) what passing in `null` here really means 🙂

Reply:
I was just following the same implementation of the `int? maxTracesPerInterval` and I think your point is also valid for `maxTracesPerInterval`.
But I can totally do it with the default value.

Reply:
I was just following the same implementation of the `int? maxTracesPerInterval`.
Null values are also provided with the `TracerRateLimiter`.

But I can totally do in this way with a default value with a new TracerRateLimiter constructor.

Reply:
Changed in 2207565eaeddaa25f1df55b9341c65af2dbb1deb